### PR TITLE
Display unverified user message on create event page

### LINKED ISSUE #2312: Warning to verify Email
**Describe the bug**
There should be a global warning after a user registers, if not global, there should be a message in the login form which comes after registering to verify the Email Id. For now, the email verification warning only shows in the user's profile. As unverified Users are not allowed to create an event, I think it should be global.

**Additional context**
Working on it.

### PR DESCRIPTION
**Pull request by prateekj117**
#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
An unverified User warning is now displayed on the Create Event page as well as unverified Users are not allowed to create an event. 

Fixes #2312

### CODE REVIEW

**Comment by prateekj117** [comment]
@CosmicCoder96 Please review.

File: app/templates/create.hbs [hunk 1]
```diff
@@ -1,4 +1,9 @@
 <div class="ui container">
+  <div class="ui stackable grid">
```
Comment (iamareebjamal) [inline app/templates/create.hbs]: This will unconditionally show to the user. Where's the logic here?

Comment (prateekj117) [reply app/templates/create.hbs]: @iamareebjamal It's in unverified-user-message file itself.

Comment (prateekj117) [reply app/templates/create.hbs]: @iamareebjamal It's in unverified-user-message file itself.

**Comment by iamareebjamal** [comment]
I think the entire div should be visible only if the message is there. @CosmicCoder96 will tell better

**Comment by prateekj117** [comment]
@iamareebjamal That's how it was implemented in the profile page. Also, I tried, the message doesn't display when the user is verified and does not disturb the UI.

**Comment by prateekj117** [comment]
@CosmicCoder96 Ok.

**Pull request merged**